packages/eslint-config/index.js

```diff
@@ -0,0 +1,25 @@
+// See @wordpress/prettier-config.
+const prettierConfig = {
+	useTabs: true,
+	tabWidth: 4,
+	printWidth: 100, // Default 80.
```

Comment:
Can we not just override this value?

Reply:
We could by adding `@wordpress/prettier-config` as a dependency. But it's [just one file](https://github.com/WordPress/gutenberg/blob/master/packages/prettier-config/lib/index.js)..